metricproviders/kayenta/kayenta_test.go

```diff
@@ -136,19 +136,6 @@ func TestRunSuccessfully(t *testing.T) {
 	c := NewTestClient(func(req *http.Request) *http.Response {
 		if req.URL.String() == jobURL {
 			assert.Equal(t, req.URL.String(), jobURL)
-
-			body, _ := ioutil.ReadAll(req.Body)
-			assert.Equal(t, string(body), `
-							{
-								"scopes": {
-										"default":{"controlScope": {"scope":"app=guestbook and rollouts-pod-template-hash=xxxx","region":"us-=west-2","step":60,"start":"2019-03-29T01:08:34Z","end":"2019-03-29T01:38:34Z"}, "experimentScope": {"scope":"app=guestbook and rollouts-pod-template-hash=yyyy","region":"us-=west-2","step":60,"start":"2019-03-29T01:08:34Z","end":"2019-03-29T01:38:34Z"}}
-								},
-                                "thresholds" : {
-                                    "pass": 90,
-                                    "marginal": 75
-                                }
-                            }`)
```

Comment:
We should include this test here since we are checking that the provider constructed the body correctly